File: DayFour/python/task_one.py
temp = '';
answer = [];
counter = 0;
while(counter<9):
	userInput = input("Enter x and o: ");
	userInputforsport = int(input("Enter where you want to place your input between 1 to 9: "));
	if(userInput == 'x'  or userInput == 'o'):
		if(userInput != temp):
			if(userInputforsport > 0 or userInputforsport < 100):
				temp = userInput;
				answer[userInputforsport-1].append(userInput)
				counter+=1;
count = 0
while(count<=2):
	print(f"   {answer[count]},end=''");
	count+=1

print("\n");
count = 3
while(count<=5):
	print(f"   {answer[count]},end=''");
	count+=1
print("\n");
count = 6
while(count<=8):
	print(f"    {answer[count]},end=''");
	count+=1
print("\n");

File: DayFour/js/taskone.js
const prompt = require("prompt-sync")();
let temp = '\0';
let one = 'x';
let two = 'o';
let answer = [];
for(let counter = 0; counter<9;){
let userInput = prompt("Enter x and o: ");
let userInputforsport = prompt("Enter where you want to place your input between 1 to 9: ");
if(userInput == 'x'  || userInput == 'o'){
if(userInput != temp){
if(userInputforsport > 0 || userInputforsport < 100){
temp = userInput;
answer[userInputforsport-1] = userInput;
counter++;
}
}
}
}

for(let count = 0; count<=2;count++){
console.log("    "+answer[count]);
}
console.log("\n");
for(let count = 3; count<=5;count++){
console.log("    "+answer[count]);
}
console.log("\n");
for(let count = 6; count<=8;count++){
console.log("    "+answer[count]);
}
console.log("\n");
